src/main/java/com/funnyboyroks/drawlib/renderer/ShapeRenderer.java

```diff
@@ -48,6 +48,17 @@ public void setColor(@NotNull Color color) {
         this.color = color;
     }
 
+    /**
+     * @param step_size The new step between particles. Lower values may cause performance issues on client and/or server, especially on low-end devices. Higher value may make the lines hard to see. Must be greater than 0.
+     */
+    public void setStepSize(@NotNull double step_size) {
```

Comment:
Would you remove this unnecessary `@NotNull`?

Reply:
Hmm, I didn't read correctly your previous change request. However I don't know when I added that... Change should be done tonight, if not it will be on tuesday.

Reply:
Okay, no worries!  I'd just assumed you had copied one of the other set methods and just kept that by mistake.